frontend/catalyst/api_extensions/__init__.py

```diff
@@ -42,6 +42,10 @@
     ctrl,
     measure,
 )
+from catalyst.api_extensions.quantum_passes import (
+    cancel_inverses,
+    inject_transform_named_sequence,
```

Comment:
perhaps this injection of transform_named_sequence makes more sense as a internal dev function, not a top level api

Reply:
done